package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "surveys",
-  "version": "1.0.0",
+  "version": "0.1.3",
```

Comment:
Please review changes to all files that will go into a git commit.  Don't commit pre-release extension versions that will end up in master.  You can select which files (or even parts of files) will go into a commit.